src/ui/utils/prefs.ts

```diff
@@ -21,6 +21,8 @@ pref("devtools.consoleFilterDrawerExpanded", true);
 pref("devtools.hitCounts", "hide-counts");
 
 // app features
+pref("devtools.features.basicProcessingLoadingBar", false);
+pref("devtools.features.enableQueryCache", false);
```

Comment:
Where did "enableQueryCache" come from? 🤨 Was it missing before? It doesn't seem related to this PR.

Reply:
Huh. It was (and still is) also on line 30 🤔  Good catch!